skynet/dgpu/compute.py

```diff
@@ -81,153 +69,140 @@ def prepare_params_for_diffuse(
         _params
     )
 
+_model_name: str = ''
+_model_mode: str = ''
+_model = None
 
-# TODO, yet again - drop the redundant prefix ;)
-class SkynetMM:
-    '''
-    (AI algo) Model manager for loading models, computing outputs,
-    checking load state, and unloading when no-longer-needed/finished.
+@cm
+def maybe_load_model(name: str, mode: str):
```

Comment:
typing for what is `yielded` would also be handy here, even if we just use a [type-alias](https://typing.readthedocs.io/en/latest/spec/aliases.html) above on the `_model` global 😎

Reply:
Done in `structify` branch